packages/zowe-explorer/package.json

```diff
@@ -1768,7 +1768,7 @@
     ]
   },
   "scripts": {
-    "build": "pnpm copy-secrets && pnpm clean:bundle && pnpm license && webpack --mode development && pnpm madge",
+    "build": "pnpm copy-secrets && pnpm clean:bundle && pnpm license  && pnpm build:nls && webpack --mode development && pnpm madge",
```

Comment:
not requesting changes, but there's an extra space after `pnpm license`

Reply:
@traeok can you explain the issue seen please? was it brought in with previous changes or pre-existing with 3.0.1? I will push a fix for the extra space but if the bug is existing in 3.0.1 it should wait until the 3.0.3 patch release.